whipper/common/program.py

```diff
@@ -474,10 +474,12 @@ def getCoverArt(self, path, cover_art, release_id):
         """
         Get cover art image from Cover Art Archive if user chooses to
 
-        :type  release_id: str
```

Comment:
Boolean type in python is named `bool` :wink:

Reply:
rotijgriohrtoihiorgoijtiontohejrhtorihre :DD.